google-cloud-tools-plugin/src/com/google/gct/idea/settings/ImportSettings.java

```diff
@@ -55,6 +55,7 @@
    */
   public static void doImport(String path) {
     final File saveFile = new File(path);
+    ZipFile saveZipFile = null;
```

Comment:
doh. yeah you're doing it right. Once we move away from java 6 we can use try with resources.